lib/kits/core-ui/components/common/offer-overview.ts

```diff
@@ -224,7 +238,6 @@ const OfferOverview = ({
         const {localeCode, currency, priceFrom} = translations;
         const {offer: incitoOffer} = await fetchOffer({viewId, publicationId});
         offer = incitoOffer;
-        offer.products = transformProducts(offer, products);
```

Comment:
Not needed anymore?

Reply:
Yes, it will be set in the if-statement just below this line 🙂

Reply:
Only if >1 (not >0 as I would assume).

Reply:
Ah yes, otherwise, it will be set in else statement: https://github.com/tjek/tjek-js-sdk/blob/a474b80fa54aa9eca1eb8e1eab62a1b8819d1e82/lib/kits/core-ui/components/common/offer-overview.ts#L244

Sorry, I was supposed to type if-else statement 😅